package-lock.json

```diff
@@ -3145,12 +3145,14 @@
         "balanced-match": {
           "version": "1.0.0",
           "bundled": true,
-          "dev": true
+          "dev": true,
+          "optional": true
```

Comment:
@lilithxxx, Is this an expected change?

Reply:
Found this as the reason for  this unexpected changes: https://github.com/npm/npm/issues/17722

Reply:
No this file got committed by mistake. Reverted the changes in PR #6711